public/index.html

```diff
@@ -9,15 +9,20 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link
-    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400&family=Open+Sans:wght@400;600;700&display=swap"
-    rel="stylesheet" />
+      rel="apple-touch-icon"
+      href="%PUBLIC_URL%/../../src/assets/img/arkhn-logo.svg"
```

Comment:
waird that we have to use a relative path here.. is this normal ? is it resolved at compilation time (eg: when running yarn build)?